hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketCreateRequest.java

```diff
@@ -460,9 +460,10 @@ public static OMRequest handleCreateBucketWithBucketLayoutDuringPreFinalize(
    * they do not understand.
    */
   @RequestFeatureValidator(
-      conditions = ValidationCondition.OLDER_CLIENT_REQUESTS,
+      conditions = {},
       processingPhase = RequestProcessingPhase.PRE_PROCESS,
-      requestType = Type.CreateBucket
+      requestType = Type.CreateBucket,
+      maxClientVersion =  ClientVersion.ERASURE_CODING_SUPPORT
```

Comment:
I would add `{}` as default value here:

https://github.com/apache/ozone/blob/bde95f71a58aad4814c8313cdd9f368cdbf79038/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/validation/RequestFeatureValidator.java#L85

and tweak changes in `@RequestFeatureValidator` usage:

```diff
-      conditions = ValidationCondition.OLDER_CLIENT_REQUESTS,
+      maxClientVersion = ClientVersion.ERASURE_CODING_SUPPORT,
```

to reduce the patch.  This changes only 1 line per item, instead of 3 lines + an unchanged line in the middle.